c/tskit/tables.c

```diff
@@ -10026,7 +10026,20 @@ tsk_table_collection_union(tsk_table_collection_t *self,
     memset(individual_map, 0xff, other->individuals.num_rows * sizeof(*individual_map));
     memset(population_map, 0xff, other->populations.num_rows * sizeof(*population_map));
     memset(site_map, 0xff, other->sites.num_rows * sizeof(*site_map));
-
+    for (k = 0; k < (tsk_id_t) other->individuals.num_rows; k++) {
+        individual_map[k] = TSK_NULL;
```

Comment:
isn't this taken care of by the `memset` above?

Reply:
Ah of course!